tests/sql/count.slt

```diff
@@ -0,0 +1,23 @@
+statement ok
+create table t(v int)
+
+statement ok
+insert into t values (1), (2), (3), (4), (5), (6), (7), (8)
+
+query I
+select count(*) from t
+----
+8
+
+query I
+select count(*) as 'total' from t where v > 5
+----
+3
+
+statement ok
+delete from t where v = 7
+
+query I
+select count(*) from t where v > 5
```

Comment:
The plan doesn't look good, but we can save it for another PR...

```
> explain select count(*) from t where v > 5
PhysicalProjection: exprs [InputRef #0]
  PhysicalSimpleAgg: 1 agg calls
    PhysicalTableScan: table #8, columns [1, 0], with_row_handler: false, is_sorted: false, expr: Gt(InputRef #0, Int32(5) (const))
```

We should only scan one column for this query.

Reply:
```
PhysicalProjection: exprs [InputRef #0]
  PhysicalSimpleAgg: 1 agg calls
    PhysicalTableScan: table #9, columns [0], with_row_handler: true, is_sorted: false, expr: Gt(InputRef #0, Int32(5) (const))
```

The plan is not optimal. As we already have column 0 scanned, we don't need row handler.

We can refine the binder logic later to handle this case, and get this PR merged at first. Thanks!

Reply:
(Maybe this should better be a task for optimizer? Only optimizer know how many columns will be scanned indeed...) Let's discuss in https://github.com/risinglightdb/risinglight/issues/482 cc @st1page